### PR TITLE
Add compatibility with Stairs, Moreblocks

### LINKED ISSUE #13: Add support for moreblocks saw
Currently mithrill, silver and tin blocks are not supported in saw.

### PR DESCRIPTION
**Pull request by Jat15**
Update sound. 
Fixes #13.

### CODE REVIEW

**Comment by Desour** [comment]
You should write here to first comment `Fixes #13.`, then it will be closed automatically by the merger on merge.
Code looks good.

**Comment by Jat15** [comment]
@DS-Minetest written in commentary.

**Comment by thomasrudin** [comment]
Any progress on this? Looks like a simple addition... :confused:

File: locale/es.txt [hunk 1]
```diff
@@ -6,6 +6,8 @@
 %s Lump = Pepita de %s
 %s Ingot = Lingote de %s
 %s Block = Bloque de %s
+%s Block Slab =
```
Comment (Panquesito7) [inline locale/es.txt]:
```suggestion
%s Block Slab = Losa de bloque de %s
```

File: locale/es.txt [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+%s Block Stair =
```
Comment (Panquesito7) [inline locale/es.txt]:
```suggestion
%s Block Stair = Escalera de bloque de %s
```

File: init.lua [hunk 3]
```diff
@@ -118,8 +118,26 @@ local function add_ore(modname, description, mineral_name, oredef)
...
 		})
+		if minetest.get_modpath("moreblocks") then	
+			stairsplus:register_all("moreores", mineral_name.."_block", block_item, {
+				description = S("%s Block"):format(S(description)),
+				tiles = { img_base .. "_block.png" },
+				groups = {snappy = 1, bendy = 2, cracky = 1, melty = 2, level= 2},
+				sounds = default.node_sound_metal_defaults()
+			})
+		elseif minetest.get_modpath("stairs") then
+			stairs.register_stair_and_slab(
```
Comment (louisroyer) [inline init.lua:116]: When minetest/minetest_game/pull/2584 will be merged, `desc_stair_outer` and `desc_stair_inner` parameters could be added (with translations for `Outer @1 Stair` and `Inner @1 Stair` in `locale` directory)

File: locale/fr.txt [hunk 4]
```diff
@@ -6,6 +6,8 @@
 %s Lump = Roche en %s
 %s Ingot = Lingot en %s
 %s Block = Bloc en %s
+%s Block Slab = Demi-dalle en %s
```
Comment (louisroyer) [inline locale/fr.txt]:
```suggestion
%s Block Slab = Dalle en %s
```
to be consistent with others slabs defined in `stairs` mod.

File: locale/fr.txt [hunk 5]
```diff
@@ -6,6 +6,8 @@
...
+%s Block Stair = Escaliers en %s
```
Comment (louisroyer) [inline locale/fr.txt]:
```suggestion
%s Block Stair = Escalier en %s
```
to be consistent with others stairs defined in `stairs`mod.

File: depends.txt [hunk 6]
```diff
@@ -1,2 +1,4 @@
 default
 mg?
+moreblocks?
```
Comment (louisroyer) [inline depends.txt]: Replace lines 3 and 4 by
```
optional_depends = moreblocks, stairs
```
in `mod.conf`

File: init.lua [hunk 7]
```diff
@@ -118,8 +118,26 @@ local function add_ore(modname, description, mineral_name, oredef)
...
+				mineral_name.."_block",
+				block_item,
+				{snappy = 1, bendy = 2, cracky = 1, melty = 2, level= 2},
+				{ img_base .. "_block.png"},
+				S("%s Block Stair"):format(S(description)),
```
Comment (louisroyer) [inline init.lua]:
```suggestion
				S("@1 Block Stair", S(description)),
```

File: init.lua [hunk 8]
```diff
@@ -118,8 +118,26 @@ local function add_ore(modname, description, mineral_name, oredef)
...
+				S("%s Block Slab"):format(S(description)),
```
Comment (louisroyer) [inline init.lua]:
```suggestion
				S("@1 Block Slab", S(description)),
```

**Review by louisroyer — CHANGES_REQUESTED** [review]
Also, you should rebase on master HEAD and translations you added should be moved into `*.tr` files (with some syntax changes).

**Review by Panquesito7 — APPROVED** [review]
Didn't test; looks good, though. I believe [More Ores](https://github.com/minetest-mods/moreores) should be updated, too.

**Comment by GreenDirkfried** [comment]
I tested this in my own fork (https://github.com/GreenDirkfried/moreores/pull/5/files) and got undefined nodes after proceeding like follows:
I started a world with the "stairs" mod included and took and placed some (mithril) stairs.
Then I closed the world and added the "moreblocks" mod, which leads to the old stairs being now undefined.
Can you confirm this problem?
I guess there should be an alias included.... @Jat15

**Pull request closed without merging**